pom.xml

```diff
@@ -51,15 +51,16 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-enforcer-plugin</artifactId>
-          <version>3.4.1</version>
+          <version>3.5.0</version>
```

Comment:
In the cms we are on 3.4.1 afaik. But IIUC this is only an aggregator and the actual modules that contain dependencies / code do have the parents from the blueprint, so I wonder if we need this plugin here at all.

Reply:
At least partly, this repository can be used outside Blueprint-scope (like running the contract tests). Thus, adding some extra rule here, seems to be fine and does not collide with Blueprint.

Keeping it as is.